manual/manual_circuit.rst

```diff
@@ -817,6 +817,48 @@ The :py:class:`CircBox` construction is good for subroutines where the instructi
     print(circ.get_commands())
     print(circ.free_symbols())
 
+Clifford Tableaux
+=================
+
+The Clifford (a.k.a. stabilizer) fragment of quantum theory is known to exhibit efficient classical representations of states and unitaries. This allows for short descriptions that can fully characterise a state/unitary and efficient circuit simulation. Whilst the Clifford group can be characterised as the operations generated by `CX`, `H`, and `S` gates with qubit initialisation in the :math:`|0\rangle` state, it is also the largest group of operations under which the Pauli group is closed, i.e. for any tensor of Paulis :math:`P` and Clifford operation :math:`C`, :math:`CPC^\dagger` is also a tensor of Paulis.
+
+Any state in the Clifford fragment is uniquely identified by those tensors of Pauli operators that stabilize it (those :math:`P` such that :math:`P|\psi\rangle = |\psi\rangle`). These stabilizers form a group of size :math:`2^n` for an :math:`n` qubit state, but it is sufficient to identify :math:`n` independent generators to specify the group. If a Clifford gate :math:`C` is applied to the state, each generator :math:`P` can be updated to :math:`P' = CPC^\dagger` since :math:`C|\psi\rangle = CP|\psi\rangle = (CPC^\dagger)C|\psi\rangle`. We can therefore characterise each Clifford operation by its actions on generators of the Pauli group, giving us the Clifford tableau form. In ``pytket``, the :py:class:`UnitaryTableau` class uses the binary symplectic representation from Aaronson and Gottesman [Aaro2004]_.
+
+.. jupyter-execute::
+
+    from pytket.circuit import OpType, Qubit
+    from pytket.tableau import UnitaryTableau
+    tab = UnitaryTableau(3)
+    tab.apply_gate_at_end(OpType.S, [Qubit(0)])
+    tab.apply_gate_at_end(OpType.CX, [Qubit(1), Qubit(2)])
+    print(tab)
+
+The way to interpret this format is that, for example, the top rows state that the unitary transforms :math:`X_0 I_1 I_2` at its input to :math:`-Y_0 I_1 I_2` at its output, and it transforms :math:`I_0 X_1 I_2` to :math:`I_0 X_1 X_2`.
+
+The primary use for tableau in ``pytket`` is as a scalable means of specifying a Clifford unitary for insertion into a circuit as a Box. This can then be decomposed into basic gates during compilation.
```

Comment:
```suggestion
The primary use for tableaux in ``pytket`` is as a scalable means of specifying a Clifford unitary for insertion into a circuit as a Box. This can then be decomposed into basic gates during compilation.
```